### ISSUE TITLE
RStudio freezing after evaluate()

### ISSUE THREAD

**Issue by fullmetalomelette** [rstudio/keras#1035]
I believe this is the same issue that was reported [here](https://github.com/rstudio/keras/issues/957), which was closed without resolution.

Running evaluate() on certain trained keras/TF models (example below) within RStudio results in QtWebEngineProc going to 100% and the system freezing for a solid ~5-10 minutes before finally returning the performance metric, as well as printing equal signs that fill up the entire console. When I run the exact same command in R on the cmd-line there is no freezing/lag and the evaluation executes immediately with no printing of equal signs.

I am using keras 2.2.5.0 with tensorflow 2.0.0 (CPU version).

Reproducible example:
```
library(keras)
library(tidyverse)

max_features <- 10000                                         
maxlen <- 20                                                  
imdb <- dataset_imdb(num_words = max_features)
c(c(x_train, y_train), c(x_test, y_test)) %<-% imdb           
x_train <- pad_sequences(x_train, maxlen = maxlen) # Restrict each entry to the first 20 words    
x_test <- pad_sequences(x_test, maxlen = maxlen)

model <- keras_model_sequential() %>%
  layer_embedding(input_dim = 10000, output_dim = 8,            
                  input_length = maxlen) %>%
  layer_flatten() %>%                                           
  layer_dense(units = 1, activation = "sigmoid")                
model %>% compile(
  optimizer = "rmsprop",
  loss = "binary_crossentropy",
  metrics = c("acc")
)
summary(model)
history <- model %>% fit(
  x_train, y_train,
  epochs = 10,
  batch_size = 32,
  validation_split = 0.2
)

model %>% evaluate(x_test, y_test)
```

Session info:
```
> sessionInfo()
R version 3.6.0 (2019-04-26)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Mojave 10.14.6

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] forcats_0.4.0   stringr_1.4.0   dplyr_0.8.5     purrr_0.3.3     readr_1.3.1    
 [6] tidyr_1.0.0     tibble_2.1.3    ggplot2_3.3.0   tidyverse_1.2.1 keras_2.2.5.0  

loaded via a namespace (and not attached):
 [1] reticulate_1.14    tidyselect_1.0.0   haven_2.2.0        lattice_0.20-38   
 [5] colorspace_1.4-1   vctrs_0.2.4        generics_0.0.2     getopt_1.20.3     
 [9] base64enc_0.1-3    blob_1.1.1         rlang_0.4.5        pillar_1.4.2      
[13] withr_2.1.2        optparse_1.6.2     glue_1.3.1         DBI_1.0.0         
[17] tidydataclient_1.0 bit64_0.9-7        modelr_0.1.4       readxl_1.3.1      
[21] lifecycle_0.1.0    tensorflow_2.0.0   munsell_0.5.0      gtable_0.3.0      
[25] cellranger_1.1.0   rvest_0.3.4        memoise_1.1.0      tfruns_1.4        
[29] broom_0.5.2        logging_0.9-107    Rcpp_1.0.1         backports_1.1.4   
[33] scales_1.1.0       jsonlite_1.6       bit_1.1-14         hms_0.5.3         
[37] packrat_0.5.0      digest_0.6.24      stringi_1.4.3      grid_3.6.0        
[41] cli_1.1.0          fasttime_1.0-2     tools_3.6.0        magrittr_1.5      
[45] RSQLite_2.1.1      tidydatar_1.0      crayon_1.3.4       whisker_0.3-2     
[49] pkgconfig_2.0.2    zeallot_0.1.0      xml2_1.2.0         lubridate_1.7.4   
[53] assertthat_0.2.1   httr_1.4.1         rstudioapi_0.10    R6_2.4.1          
[57] nlme_3.1-140       compiler_3.6.0
```

**Comment by henry090** [comment]
Hi, This is my session info. And there is no such freezing. I suggest to download the latest version of RStudio. My current version is ```Version 1.2.5042``` Link : https://download1.rstudio.org/desktop/macos/RStudio-1.2.5042.dmg
```
sessionInfo()
R version 3.6.3 (2020-02-29)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.4

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] forcats_0.5.0   stringr_1.4.0   dplyr_0.8.5     purrr_0.3.4     readr_1.3.1     tidyr_1.0.2     tibble_3.0.1   
 [8] ggplot2_3.3.0   tidyverse_1.3.0 keras_2.2.5.0  

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.4.6     cellranger_1.1.0 pillar_1.4.3     compiler_3.6.3   dbplyr_1.4.2     base64enc_0.1-3  tools_3.6.3     
 [8] zeallot_0.1.0    lubridate_1.7.8  jsonlite_1.6.1   lifecycle_0.2.0  nlme_3.1-144     gtable_0.3.0     lattice_0.20-38 
[15] pkgconfig_2.0.3  rlang_0.4.5      reprex_0.3.0     Matrix_1.2-18    cli_2.0.2        DBI_1.1.0        rstudioapi_0.11 
[22] haven_2.2.0      withr_2.2.0      xml2_1.3.1       httr_1.4.1       fs_1.4.1         hms_0.5.3        generics_0.0.2  
[29] vctrs_0.2.4      grid_3.6.3       tidyselect_1.0.0 reticulate_1.15  glue_1.4.0       R6_2.4.1         fansi_0.4.1     
[36] readxl_1.3.1     modelr_0.1.6     magrittr_1.5     whisker_0.4      scales_1.1.0     backports_1.1.6  tfruns_1.4      
[43] ellipsis_0.3.0   rvest_0.3.5      assertthat_0.2.1 colorspace_1.4-1 tensorflow_2.0.0 stringi_1.4.6    munsell_0.5.0   
[50] broom_0.5.6      crayon_1.3.4    
```

**Comment by fullmetalomelette** [comment]
Thanks for the suggestion. I am also using RStudio `1.2.5042` (latest) so that's not the issue here.

**Comment by fullmetalomelette** [comment]
Is there a recommended way to remove Keras and TensorFlow installation completely and reinstall from scratch? I assume I just need to delete the associated anaconda environment and remove the R package?

**Comment by henry090** [comment]
From Mac OS Terminal you could type:
```
# reticulate::py_config() # example
/Users/turgutabdullayev/Library/r-miniconda/envs/r-reticulate/bin/python -m pip uninstall keras tensorflow
```

**Comment by fullmetalomelette** [comment]
Thanks @henry090 

Fresh install of keras and tensorflow seems to have fixed this issue.
